lib/slide_rule/version.rb

```diff
@@ -1,3 +1,3 @@
 module SlideRule
-  VERSION = '0.2.1'
+  VERSION = '0.2.1'.freeze
```

Comment:
It is a cop, but we don't do that in our system. I added that to the ignore and meant to remove this. I can change it though.